manage_breast_screening/participants/models.py

```diff
@@ -10,43 +10,69 @@
 
 logger = getLogger(__name__)
 
+
 # List of ethnic groups from
 # https://design-system.service.gov.uk/patterns/equality-information/
 # This list is specific to England.
-ETHNIC_GROUPS = {
-    "White": [
-        "English, Welsh, Scottish, Northern Irish or British",
-        "Irish",
-        "Gypsy or Irish Traveller",
-        "Any other White background",
-    ],
-    "Mixed or multiple ethnic groups": [
-        "White and Black Caribbean",
-        "White and Black African",
-        "White and Asian",
-        "Any other mixed or multiple ethnic background",
-    ],
-    "Asian or Asian British": [
-        "Indian",
-        "Pakistani",
-        "Bangladeshi",
-        "Chinese",
-        "Any other Asian background",
-    ],
-    "Black, African, Caribbean or Black British": [
-        "African",
-        "Caribbean",
-        "Any other Black, African or Caribbean background",
-    ],
-    "Other ethnic group": ["Arab", "Any other ethnic group"],
-}
+class Ethnicity:
+    # fmt: off
+    DATA = {
+        "White": [
+            { "id": "english_welsh_scottish_ni_british", "display_name": "English, Welsh, Scottish, Northern Irish or British", },
```

Comment:
Rather than inventing our own codes here, I think it would be useful to reuse an existing standard if possible, so that data linkage is easier.

The ONS dataset [numbers them](https://www.ons.gov.uk/methodology/classificationsandstandards/measuringequality/ethnicgroupnationalidentityandreligion) so I suggest introducing an enum, with the values being those numbers, and then if we do need to reference specific ones in the code or tests, we won't have problems with magic strings.

Reply:
I think it's a good suggestion. Spiking this locally though, it does add a layer of obfuscation that makes tests and debugging less clear and more error-prone. It's quite useful to be able to see the actual ethnicity (rather than its number) in tests and database records. The snake cased strings are also easier to grep/autocomplete than the display names (or numerical identifiers). We could retain some of this by having the enum map the ONS numbers to the snake cased identifiers, but then we need to add a mapping between identifier and display name ☹️ .

The ethnicity descriptions are unlikely to change, certainly not frequently. We can also easily add the ONS identifiers to this data structure at the point we actually need them, giving us a mapping.

WDYT?